src/relayers/pimlico/index.ts

```diff
@@ -1,118 +0,0 @@
-import 'dotenv/config';
```

Comment:
@Akhilleshgoswami We still use Pimlico relayers. Please leave Pimlico logic as it is and remove the rest from relayers